releases/release-1.20/release-team.md

```diff
@@ -0,0 +1,16 @@
+# Kubernetes 1.20 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
+|---|---|---|
+| Lead | Jeremy Rickard ([@jeremyrickard](https://github.com/jeremyrickard) / Slack: `@onlydole`) | Daniel Mangum ([@hasheddan](https://github.com/hasheddan), Nabarun Pal ([@palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`), Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) |
```

Comment:
1. Your Slack username needs to be fixed.
2. Close the parenthesis on Savitha's username

Reply:
Oops 👍

Reply:
Bad copy/paste from the previous form dropped Dan and Savitha's slack handles. Fixing.